src/solvers/dgsem_tree/indicators_2d.jl

```diff
@@ -223,5 +223,378 @@ function (indicator_max::IndicatorMax)(u::AbstractArray{<:Any,4},
   return alpha
```

Comment:
General note: please consider all comments left here also to for the 1D file where applicable.